integration/container/remove_test.go

```diff
@@ -0,0 +1,102 @@
+package container // import "github.com/docker/docker/integration/container"
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/integration/internal/container"
+	"github.com/docker/docker/integration/internal/request"
+	"github.com/docker/docker/internal/testutil"
+	"github.com/gotestyourself/gotestyourself/fs"
+	"github.com/gotestyourself/gotestyourself/poll"
+	"github.com/gotestyourself/gotestyourself/skip"
+	"github.com/stretchr/testify/require"
+)
+
+func getPrefixAndSlashFromDaemonPlatform() (prefix, slash string) {
+	if testEnv.OSType == "windows" {
+		return "c:", `\`
+	}
+	return "", "/"
+}
+
+func TestRemoveContainerWithRemovedVolume(t *testing.T) {
+	skip.If(t, !testEnv.IsLocalDaemon())
+
+	defer setupTest(t)()
+	ctx := context.Background()
+	client := request.NewAPIClient(t)
+
+	prefix, slash := getPrefixAndSlashFromDaemonPlatform()
+
+	tempDir := fs.NewDir(t, "test-rm-container-with-removed-volume-", fs.WithMode(0755))
+	defer tempDir.Remove()
+
+	container.Run(t, ctx, client, container.WithName("losemyvolumes"), container.WithCmd("true"), func(c *container.TestContainerConfig) {
```

Comment:
I think we should try to not use container name in most of the tests ; this will allow then to "probably" run then in parallel later 👼

Reply:
Thanks @vdemeester. Updated.